### PR TITLE
Adding Login-Form

### PR DESCRIPTION
**Pull request by anshulgupta222**
Hi Harsh ,

Please review this component

### CODE REVIEW

File: src/app/login-form/login-form.component.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<div class="row justify-content-center">
+  <div class="col-md-3 border">
+    <div class="text-center my-2" style="font-weight: bold">Login</div>
```
Comment (harshdave71720) [inline src/app/login-form/login-form.component.html:3]: find bootstrap class for bold text

File: src/app/login-form/login-form.component.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    <form [formGroup]="form">
+      <div class="form-group">
+        <div class="row">
+          <label for="email" class="form-label">Email</label>
+          <div class="col-md-12">
+            <input
+              formControlName="email"
+              id="email"
+              type="text"
```
Comment (harshdave71720) [inline src/app/login-form/login-form.component.html:12]: change the type to email

File: src/app/login-form/login-form.component.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        </div>
+      </div>
+      <div class="form-group">
+        <div class="row">
+          <label for="password" class="form-label">Password</label>
+          <div class="col-md-12">
+            <input
+              formControlName="password"
+              id="password"
+              type="text"
```
Comment (harshdave71720) [inline src/app/login-form/login-form.component.html:36]: change type to password

File: src/app/login-form/login-form.component.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  templateUrl: './login-form.component.html',
+  styleUrls: ['./login-form.component.css']
+})
+export class LoginFormComponent{
+
+  form = new FormGroup({
+    email:new FormControl('',[Validators.required,Validators.email]),
+    password: new FormControl('',Validators.required)
+  });
+
```
Comment (harshdave71720) [inline src/app/login-form/login-form.component.ts:15]: add empty method for form submission

**Review by harshdave71720 — CHANGES_REQUESTED** [review]
Hello Anshul, Please complete the suggested changes

File: src/app/login-form/login-form.component.html (hunk 1 above)
Comment (anshulgupta222) [reply src/app/login-form/login-form.component.html:3]: Added the bootstrap class "fw-bold"

File: src/app/login-form/login-form.component.html (hunk 2 above)
Comment (anshulgupta222) [reply src/app/login-form/login-form.component.html:12]: Added the input type "email"

File: src/app/login-form/login-form.component.ts (hunk 4 above)
Comment (anshulgupta222) [reply src/app/login-form/login-form.component.ts:15]: Added the method --> login()

File: src/app/login-form/login-form.component.html (hunk 3 above)
Comment (anshulgupta222) [reply src/app/login-form/login-form.component.html:36]: Added the input type "password"

**Review by harshdave71720 — APPROVED** [review]

**Pull request merged**